lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -3707,3 +3733,75 @@ def get_test_data(delta=0.05):
     Y = Y * 10
     Z = Z * 500
     return X, Y, Z
+
+
+class _Quaternion:
+    """
+    Quaternions
+    consisting of scalar, along 1, and vector, with components along i, j, k
+    """
```

Comment:
Please also add an `.inv()` method, as I expect we will want to invert quaternion rotations in the future.